app/assets/javascripts/reducers/stories.js

```diff
@@ -12,7 +15,47 @@ const storiesReducer = (state = initialState, action) => {
   switch (action.type) {
     case actionTypes.RECEIVE_STORIES:
       return action.data;
+    case actionTypes.CREATE_STORY:
+      if (isCreating(state)) {
+        const story = state.find(isNew);
+
+        if (!creatingInAnotherColumn(story, action.attributes.state)) {
+          return state;
+        }
+
+        return [
+          {
+            ...editStory(story, action.attributes),
```

Comment:
I'm kinda confused by what is being edited since we're creating. Can you elaborate about this part of the code?